### ISSUE TITLE
Linux Build Fails

### ISSUE THREAD

**Issue by ricpruss** [aswinmurali-io/osumffmpeg#3]
Linux build fails because the platform dir has not been checked into github.  Same is true for the other platforms except maybe snap.

**Comment by ricpruss** [comment]
Actually this seemed to work.
flutter create . --platforms=linux
flutter build linux

**Comment by aswinmurali-io** [comment]
Yeah, the goal was to generate platform directory during build time. I just didn't want to clutter the project in GitHub. Maybe I should point it out in README file. Thanks for telling me.
